.travis.yml

```diff
@@ -39,7 +39,7 @@ jobs:
       script:
         - CLUSTER_NAME="substra-tests-$(date -u +'%Y-%m-%d-%Hh%M')"
         - cd ci/
-        - python -u ./run-ci.py --keys-directory=./keys/ --cluster-name=${CLUSTER_NAME}
+        - python -u ./run-ci.py --no-cache --keys-directory=./keys/ --cluster-name=${CLUSTER_NAME}
```

Comment:
@AurelienGasser what do you think about that ?

Reply:
I had a quick look at this. 

This PR completely disables the cache for the CI, I think we can avoid doing that.

The root of the issue seems to be that the output image tag `substra-tests:ci-xxx` only depends on the SHA of the `substra-tests` commit, not the `substra` commit.

https://github.com/SubstraFoundation/substra-tests/blob/9e32fdf01c750dc67c1cf302b6d1e2bbfed3d4f9/ci/cloudbuild/substra-tests.yaml#L6

^ here `_COMMIT` refers to substra-tests.

A potential fix:

1. In `run-ci.py`, stop supporting any "ref" for substra, and only support branch names, like for the other projects (see [here](https://github.com/SubstraFoundation/substra-tests/blob/master/ci/run-ci.py#L60-L63). In other words, rename `SUBSTRA_GIT_REF` to `SUBSTRA_BRANCH`.
2. In `run-ci.py`, fetch the latest commit corresponding to the substra branch (see [here](https://github.com/SubstraFoundation/substra-tests/blob/master/ci/run-ci.py#L287-L293))
3. Pass it to `substra-tests.yaml` as a new substition parameter (see [here](https://github.com/SubstraFoundation/substra-tests/blob/master/ci/run-ci.py#L346))
4. In `substra-tests.yaml`, use the commit SHA in the image tag

```diff
- --destination=eu.gcr.io/$PROJECT_ID/substra-tests:ci-${_COMMIT}
+ --destination=eu.gcr.io/$PROJECT_ID/substra-tests:ci-${_COMMIT}-${_SUBSTRA_GIT_COMMIT}
```

Reply:
Thanks for the detailed answer ! I changed it so that the image depends on the commit of substra and substra-tests, it seems to work well